packages/arb-ts/src/lib/bridge_helpers.ts

```diff
@@ -192,16 +199,14 @@ export class BridgeHelper {
 
   static getDepositTokenEventData = async (
     l1Transaction: providers.TransactionReceipt,
-    tokenType: 'ERC20' | 'ERC777' = 'ERC20',
-    l2BridgeAddress?: string
+    l2BridgeAddress: string
```

Comment:
Is there a reason to not make the address optional here? We only need the bridge interface since we pass in the receipt. I did it this way because we can just hardcode a random value to parse logs from the receipt using the interface

Reply:
The same applies to the other methods that parse logs. There probably is a neater way to get the interface using typechain, but I wasn't able to figure it out

Reply:
We can deal with this in a subsequent PR, as noted in https://github.com/OffchainLabs/arbitrum/pull/995/files#r605180872

Reply:
subsequent PRs 👍